### ISSUE TITLE
Captcha not showing

### ISSUE THREAD

**Issue by youhosi** [akhoury/nodebb-plugin-spam-be-gone#67]
Logs:

```
scripts.js:7 Uncaught ReferenceError: pluginName is not defined
    at n (scripts.js:7)
    at i (scripts.js:16)
    at scripts.js:86
    at dispatch (jquery.js:5183)
    at m.handle (jquery.js:4991)
    at Object.trigger (jquery.js:8249)
    at jquery.js:8327
    at Function.each (jquery.js:354)
    at x.fn.init.each (jquery.js:189)
    at x.fn.init.trigger (jquery.js:8326)
```

```
if (window.plugin && plugin[pluginName] && plugin[pluginName].recaptchaArgs && $('#' + plugin[pluginName].recaptchaArgs.targetId).length) {
```

**Comment by akhoury** [comment]
https://github.com/akhoury/nodebb-plugin-spam-be-gone/commit/0872198eddd3f9e1bfc137be2e747ea8c58cca5a

**Comment by akhoury** [comment]
sorry :/ published 0.6.1
